Added Website Updates Guide (#184)
o3de/community: process-docs-and-markdown/website-updates.md

            </a>
          </div>
          <a data-target="#carouselPartnersIndicators" data-slide="next" class="btn">
            <i class="fas fa-chevron-right"></i>
          </a>
        </div>
      </div>

    </div>
  </div>
</section>
```

If we are to take this section, it would represent slider 2

```html
          <div class="carousel-item">
            <div class="partner-denomination">Premier Partners</div>
			<div class="partner-logo oppo-logo"></div>
			<div class="partner-denomination">General Partners</div>
            <div class="partner-logo audiokinetic-logo"></div>
            <div class="partner-logo carbonated-logo"></div>
            <div class="partner-logo futurewei-logo"></div>
          </div>
```
![image](/process-docs-and-markdown/files/media/member-image.png)

Say we wanted to add more logos and we do not have any more room. To add another slider, you would create the following. 

```html
	            <div class="carousel-item">
          </div>
```

Followed by 

```html
	     <a data-target="#carouselPartnersIndicators" data-slide-to="5" class="btn carousel-btn">
              <i class="fas fa-circle"></i>
            </a>
```

> And if we wanted to add the Logo, we would simply add

Reviewer: ```suggestion
To add a logo to a slider, copy and paste the following into the new carousel item `div`:
```

```html
			<div class="partner-logo caict-logo"></div>
```

> In this case we would replace `caict-logo` with the logo name we did in the above steps

Reviewer: ```suggestion
In this example, replace `caict-logo` with your logo name.
```

> The file would then look like:

Reviewer: ```suggestion
The following version of `layouts\partials\home\partners.html` shows all of these changes:
```

              <i class="fas fa-circle"></i>
            </a>
            <a data-target="#carouselPartnersIndicators" data-slide-to="1" class="btn carousel-btn">
              <i class="fas fa-circle"></i>
            </a>
            <a data-target="#carouselPartnersIndicators" data-slide-to="2" class="btn carousel-btn">
              <i class="fas fa-circle"></i>
            </a>
            <a data-target="#carouselPartnersIndicators" data-slide-to="3" class="btn carousel-btn">
              <i class="fas fa-circle"></i>
            </a>
	     <a data-target="#carouselPartnersIndicators" data-slide-to="4" class="btn carousel-btn">
              <i class="fas fa-circle"></i>
            </a>
	     <a data-target="#carouselPartnersIndicators" data-slide-to="5" class="btn carousel-btn">
              <i class="fas fa-circle"></i>
            </a>
          </div>
          <a data-target="#carouselPartnersIndicators" data-slide="next" class="btn">
            <i class="fas fa-chevron-right"></i>
          </a>
        </div>
      </div>

    </div>
  </div>
</section>
```

## Image Updates

> To ensure that the website loads quickly, it's important to use images that are not too large in size. All images should be stored within the `/static/images/` folder. If you are adding images for a new event or series, I suggest creating a new folder to keep everything organized.

Reviewer: ```suggestion
To ensure that the website loads quickly, it's important to use images that are not too large in size. All images should be stored within the `/static/images/` folder. If you are adding images for a new event or series, create a new folder to keep everything organized.
```

> However, if the event happens again or the image is relevant to something else, you can keep those images in their original folder. But if the event is over and you remove the message, post, or banner associated with it, you should also remove the images to prevent stale images from cluttering the site.

Reviewer: ```suggestion
If the event repeats or the image is relevant to something else, you can keep the images in their original folder. But if the event is completely finished and you remove the message, post, or banner associated with it, you should also remove the images to prevent stale images from cluttering the site.
```

    </div>
    <div class="banner-call-to-action">REGISTER</div>
  </div>
</a> 
```

Will render:
![image](/process-docs-and-markdown/files/media/banner-image.png)

To edit the colors, these can be accessed in `assets/sass/custom.sass`

## Color Codes

To incorporate a new color to the website, the existing colors can be found in the website folder under `assets/sass` in the `custom.sass` file. The color codes are available around line 70 in a section. This section makes it easier to reference the color code instead of looking up the Hex code.

To add a new color, insert a new line under the `// Variables` section. For example:

If the existing section looks like this:

```php
// Variables 

$font-family-headers: "Open Sans" 
$font-family-Poppins: "Poppins", sans-serif;
$o3de-black: #000
$o3de-white: #fff
$o3de-lightgray: #ddd
$o3de-blue: #1E70EB
$o3de-darkblue: #0F3876
$o3de-docsblue: #2A4875
$o3de-mediumblue: #4B8DEF
$o3de-lightblue: #A5C6F7
$o3de-blue-highlight: #94D2FF
$o3de-green: #58BD61
$o3de-purple: #700AC7
$o3de-aqua: #0AC1C7
```

Add a new line at the bottom and name it, such as `O3DE Pink`. Then, insert the color code for the new color.

> ```php
> // Variables 
> 
> $font-family-headers: "Open Sans" 
> $font-family-Poppins: "Poppins", sans-serif;
> $o3de-black: #000
> $o3de-white: #fff
> $o3de-lightgray: #ddd
> $o3de-blue: #1E70EB
> $o3de-darkblue: #0F3876
> $o3de-docsblue: #2A4875
> $o3de-mediumblue: #4B8DEF
> $o3de-lightblue: #A5C6F7
> $o3de-blue-highlight: #94D2FF
> $o3de-green: #58BD61
> $o3de-purple: #700AC7
> $o3de-aqua: #0AC1C7
> $o3de-pink: #f16dbb
> ```
> 
> As demonstrated, `$o3de-pink: #f16dbb` has been added to the code block. Consequently, one can use o3de-pink while designing, instead of using the Hex code.

Reviewer: ```suggestion
The following code snippet shows the variables section after adding $o3de-pink:

```php
// Variables 

$font-family-headers: "Open Sans" 
$font-family-Poppins: "Poppins", sans-serif;
$o3de-black: #000
$o3de-white: #fff
$o3de-lightgray: #ddd
$o3de-blue: #1E70EB
$o3de-darkblue: #0F3876
$o3de-docsblue: #2A4875
$o3de-mediumblue: #4B8DEF
$o3de-lightblue: #A5C6F7
$o3de-blue-highlight: #94D2FF
$o3de-green: #58BD61
$o3de-purple: #700AC7
$o3de-aqua: #0AC1C7
$o3de-pink: #f16dbb
```
```
Moving the last sentence to the next paragraph.
